client/components/seasonal-page.jsx

```diff
@@ -14,35 +15,53 @@ class SeasonalPage extends React.Component {
   getProduceList(name) {
     fetch(`/api/produce-in-season?seasonName=${name}`)
       .then(results => results.json())
-      .then(produce => this.setState({ produceList: produce }))
+      .then(produce => this.setState({ produceList: produce }, this.getFeaturedProduce))
       .catch(error => console.error(error.message));
   }
 
+  getFeaturedProduce() {
+    if (!this.state.produceList.length) { return; }
+    const randomIndices = [];
+    while (randomIndices.length < 3) {
```

Comment:
we could move this number to the constructor so that it's easier to find down the road not a big deal though